tools/yaml-templates/android-test.yml

```diff
@@ -58,6 +58,13 @@ steps:
       script: 'nohup pnpm start-test-app &'
       workingDirectory: '$(ClientSdkProjectDirectory)'
 
+  - task: CodeQL3000Init@0
```

Comment:
Same thing here because Android is using MacOS 15. Not sure it is ARM based or Intel based or mixed.